Core/scripts/dirac-deploy-scripts.py

```diff
@@ -1,22 +1,55 @@
 #!/usr/bin/env python
 """
 Deploy all scripts and extensions
-Options:
- * --symlink: this will create symlinks instead of wrappers
- * <python path>: you can specify the folder where your python installation should be fetched from
+
+Usage::
+
+  dirac-deploy-scripts (<options>|<cfgFile>)* ... <python path>
+
+Arguments::
+  python path:    you can specify the folder where your python installation should be fetched from
                   to replace the shebang
+
+Example::
+
+  $ dirac-deploy-scripts
+  Scripts will be deployed at /afs/in2p3.fr/home/h/hamar/DIRAC-v5r12/scripts
+  Inspecting DIRAC module
+  Inspecting EELADIRAC module
```

Comment:
This example is so old that might end up being confusing. Also, this script is not meant to be called by users (it's automatically called by dirac-install), and of course in python3 is not called at all. 

I would remove these lines and replace them with my comment above ("this script is not meant to be called by users (it's automatically called by dirac-install").

Reply:
added description and removed old command output in example